aten/src/ATen/core/type.cpp

```diff
@@ -659,8 +659,8 @@ bool ListType::isSubtypeOfExt(const TypePtr rhs_, std::ostream* why_not) const {
   return false;
 }
 
- bool TupleType::operator==(const Type& rhs) const {
-   bool typesSame =
+bool TupleType::operator==(const Type& rhs) const {
+  bool typesSame =
        compare(rhs, [](const TypePtr a, const TypePtr b) { return *a == *b; });
```

Comment:
also need to remove one space in this line?

Reply:
Reverting this change. It's unexpected formatting.